edgexfoundry/go-mod-configuration
# refactor!: Remove use of TOML package

BREAKING CHANGE: PutConfigurationToml has be rework to be PutConfigurationMap







If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-configuration/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
See [edgex-go PR](https://github.com/edgexfoundry/edgex-go/pull/4468) testing instructions

## New Dependency Instructions (If applicable)
